pkg/jsonedit/transform.go

```diff
@@ -36,7 +36,7 @@ func Transform(jsonBytes []byte, paths []string, transformFn func(unpacked inter
 		}
 
 		if err != nil {
-			// handle error
+			return nil, err
```

Comment:
As in other parts of the code, I think this also should not return but most likely just continue